_CodingChallenges/69.4-steering-evolution.md

```diff
@@ -31,6 +31,15 @@ parts:
   url: "/CodingChallenges/69.3-steering-evolution"
 - title: "Evolutionary Steering Behaviors - Part 5"
   url: "/CodingChallenges/69.5-steering-evolution"
+  
+  contributions:
+  - title: "My Evolutionary Steering Agents with predators"
+    author:
+      name: "Ben Ashworth"
+    url: "https://talkingchef.co.uk/codingTrain/genEco/"
+    
```

Comment:
If you have a link to the source code of your contribution, you can provide it with `source: your-url-here`.

Reply:
I'm afraid i don't